libs/sdk-ui-dashboard/src/model/commandHandlers/drill/drillToDashboardHandler.ts

```diff
@@ -102,31 +111,80 @@ export function* drillToDashboardHandler(
     );
 }
 
-function selectDrillingDateFilter(state: DashboardState, widget: IInsightWidget): IDateFilter {
+function numberOrStringToNumber(input: number | string): number {
+    return isString(input) ? Number.parseInt(input) : input;
+}
+
+function dashboardDateFilterByWidget(
+    filter: IDashboardDateFilter,
+    widget: Partial<IFilterableWidget>,
+): IDashboardDateFilter {
+    if (filter.dateFilter.type === "relative") {
+        return {
+            dateFilter: {
```

Comment:
We should add `newDashboardAttributeFilter` and `newDashboardDateFilter` factories, similar as we have it for `newAttributeFilter` / `newDateFilter` and use them instead.

Reply:
Reverted to `IDateFilter`